pkg/cloud/services/compute/instance.go

```diff
@@ -759,8 +767,9 @@ func (s *Service) getPort(portID string) (port *ports.Port, err error) {
 	if portID == "" {
 		return nil, fmt.Errorf("portID should be specified to get detail")
 	}
+	mc := metrics.NewMetricPrometheusContext("port", "get")
 	port, err = ports.Get(s.networkClient, portID).Extract()
-	if err != nil {
+	if mc.ObserveRequest(err) != nil {
 		if capoerrors.IsNotFound(err) {
```

Comment:
So this may not even be a error for us if OpenStack returned an `IsNotFound` error?

Reply:
Sure, you're totally right. Thanks